src/mtcp/mtcp_util.h

```diff
@@ -35,11 +35,11 @@
     mtcp_abort();                                       \
   }
 
-#ifdef DEBUG
+#ifdef LOGGING
```

Comment:
It is available:

    gcc -I../../include -I. -DLOGGING -g -O2 -DLOGGING -c -o mtcp_restart.o\
      -DPIC -fPIC -fno-stack-protector -g -O0 mtcp_restart.c

I'll update Makefile.debug.